modules/core/src/wallet.ts

```diff
@@ -1363,19 +1364,16 @@ Wallet.prototype.accelerateTransaction = function accelerateTransaction(params,
    * @param parentTxId The ID of the transaction to get the full hex of
    * @returns {Bluebird<any>} The full hex for the specified transaction
    */
-  const getParentTxHex = ({ parentTxId }) => {
-    return co(function *coGetParentTxHex() {
-      const smartBitApiUrl = common.Environments[this.bitgo.getEnv()].smartBitApiBaseUrl + '/blockchain/tx/';
-      const txUrl = smartBitApiUrl + parentTxId + '/hex';
-      const result = yield request.get(txUrl);
-
-      if (!_.isBoolean(result.body.success) || !result.body.success) {
-        throw new Error('Did not successfully receive parent tx hex');
-      }
+  async function getParentTxHex({ parentTxId }: { parentTxId: string }): Promise<string> {
+    const explorerBaseUrl = common.Environments[self.bitgo.getEnv()].btcExplorerBaseUrl;
+    const result = await request.get(`${explorerBaseUrl}/tx/${parentTxId}/hex`);
 
-      return result.body.hex[0].hex;
-    }).call(this);
-  };
+    if (!result.text || !/([a-f0-9]{2})+/.test(result.text)) {
+      throw new Error('Did not successfully receive parent tx hex');
```

Comment:
It might be helpful to output what was received instead.

Reply:
Will do 👍